### DISCUSSION TITLE
Feedback: component code examples lack an import statement

### DISCUSSION THREAD

**Discussion by dprothero** [twilio-labs/paste#2415] [Ideas] [answered]
When I look at the code examples for a given component, say:

https://paste.twilio.design/components/badge

None of the examples show the `import` statement to actually import the component. I have to go all the way to the bottom of the page to find it. This would be extremely useful so I didn't have to go searching for that information somewhere else.

**Comment by dprothero** [comment]
[Here's an example](https://paste.twilio.design/components/badge#badge-with-button-functionality) of a code sample that's extremely difficult to be able to use because it has no import statements for any of the many components it's making use of.

To make use of that code, I have to go track down all of those components. :sad:

**Comment by nkrantz** [comment, marked as answer]
Hey @dprothero - this is really helpful insight and something we've considered in the past. We plan to address this by adding "open in CodeSandbox" buttons to all the code examples on the website. That way you can easily see all the imports for components being used in any example, and add new components to the examples to play around with too. [Here's the ticket](https://issues.corp.twilio.com/browse/DSYS-2019) - it's in our backlog and we unfortunately don't have an ETA on it yet because of the HAP but we'll update this thread once it's done. Thanks for the feedback!
